packages/web/src/components/track/mobile/TrackTile.tsx

```diff
@@ -111,7 +110,9 @@ const renderLockedContentOrPlayCount = ({
         />
       )
     }
-    return <LockedStatusPill locked={!hasStreamAccess} variant={variant} />
+    return (
+      <LockedStatusPill locked={!hasStreamAccess} variant={lockedContentType} />
```

Comment:
this is confusing - `variant` here is `readonly` but `variant` for `LockedStatusPill` is `gated | premium`? don't have to fix now tho.

Reply:
agreed. It's not great, but good enough for now